inst/rmarkdown/templates/wellcomeor/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,196 @@
+---
+title: Wellcome Open Research Article Template
+titlenote: The title should be detailed enough for someone to know whether the article would be of interest to them, but also concise. Please ensure the broadness and claims within the title are appropriate to the content of the article itself.
+author:
+  - name: Author Name1
+    affilliation: 1
+  - name: Author Name2
+    affilliation: 2
+address:
+  - code: 1
+    address: Address1
+  - code: 2
+    address: Address2
+abstract: |
+  Abstracts should be up to 300 words and provide a succinct summary of the article. Although the abstract should explain why the article might be interesting, care should be taken not to inappropriately over-emphasise the importance of the work described in the article. Citations should not be used in the abstract, and the use of abbreviations should be minimized. If you are writing a Research or Systematic Review article, please structure your abstract into Background, Methods, Results, and Conclusions.
+keywords: |
+  Please list up to eight keywords to help readers interested in your article find it more easily.
+
+date: "`r Sys.Date()`"
+bibliography: sample.bib
+biblio-style: "unsrtnat"
+link-citations: TRUE
+colorlinks: TRUE
+# You can download the csl file if preferred and use it locally with `csl: elsevier-vancouver.csl`
+csl: https://www.zotero.org/styles/elsevier-vancouver
+output: 
+  bookdown::pdf_book:
+    base_format: rticles::wellcomeor_article
+    citation_package: natbib
+    number_sections: FALSE
```

Comment:
Does this need to be set all the time ? If so we can change the default in the format function.

Reply:
Are you referring to the `number_sections` line? If so, I've set that to default to false in https://github.com/rstudio/rticles/pull/436/commits/4bb1beb4a38f526a4d0cf64db4949f73bbe50b40, so hopefully that resolves this concern, but please let me know if not

Reply:
It was not directly a concern. 
My question was: "Does the template requires `number_sections` to be FALSE by default ? "

If so, then we should just to it in the format function
```diff
diff --git a/R/article.R b/R/article.R
index 1f09012..ad9aa96 100644
--- a/R/article.R
+++ b/R/article.R
@@ -446,11 +446,11 @@ trb_article <- function(..., keep_tex = TRUE, citation_package = 'natbib') {
 }
 
 #' @section \code{wellcomeor_article}: Format for creating submissions to
 #' Wellcome Open Research. Adapted from \url{https://www.overleaf.com/project/6131911644b635ad3aaa4cb2}.
 #' @export
 #' @rdname article
-wellcomeor_article <- function(..., keep_tex = TRUE, citation_package = 'natbib') {
+wellcomeor_article <- function(..., number_sections = FALSE, keep_tex = TRUE, citation_package = 'natbib') {
   pdf_document_format(
-    "wellcomeor", keep_tex = keep_tex, citation_package = citation_package, ...
+    "wellcomeor", keep_tex = keep_tex, number_sections = number_sections, citation_package = citation_package, ...
   )
 }
```

and we would leave the template as the own in Pandoc
https://github.com/jgm/pandoc/blob/9cbea695c439dd04c9a670107ec729c503d5b368/data/templates/default.latex#L318-L322